CHANGELOG.md

```diff
@@ -423,3 +423,80 @@ implementation only).
 - Fix a bug in the schedule roll-out logic.
 
 - Added new day-count conventions: Act365NL, Basis30365, and Basis30E2360.
+
+## Rationale Behind Breaking Changes
```

Comment:
The changelog is a living document for the benefit of the team, I don't think it is visible / known externally.

I think we should identify a place in our docs where this belongs. Maybe @frankPreiwussDA has an idea?

Reply:
Agree, once ready I will move the release notes for each package to the docs (as usual), and maybe this section could be integrated into the Release Notes of Daml 2.8.0 or we find a place for it in the docs as well. I discussed it briefly with @frankPreiwussDA today already.

Reply:
I think we should leave the section in there, and also add them for any future breaking changes. In order to make it visible we will copy it in the docs and link the page from the release notes.

Reply:
@frankPreiwussDA As discussed, I added it to the "Current Release" section of the docs:
https://github.com/digital-asset/docs.daml.com/pull/549 
Will close this PR.